src/components/popper/PopperComponent.js

```diff
@@ -25,12 +26,12 @@ const PopperComponent = (props: Props) => {
         <Manager>
             <Reference>{({ ref }) => React.cloneElement(reference, { ref })}</Reference>
             {isOpen && (
-                <Popper placement={popperPlacement} modifiers={modifiers}>
+                <Popper placement={popperPlacement} modifiers={modifiers} positionFixed={positionFixed}>
                     {({ ref, style, placement, scheduleUpdate }) => {
                         const { style: contentStyles } = popperContent.props;
                         return React.cloneElement(popperContent, {
                             ref,
-                            style: { ...contentStyles, ...style },
+                            style: { ...style, ...contentStyles },
```

Comment:
Was there a reason the order of these was changed?

Reply:
The popper does some auto styling including adding a `top`, `left`, and `transform` properties that affect how the dropdown menu is positioned on the screen. With the dropdown menu, I want to override these only if `contentStyles` are passed in. Otherwise the popper styling would override what I pass in.